rqt_image_view/include/rqt_image_view/image_view.h

```diff
@@ -82,27 +83,31 @@ protected slots:
   virtual QSet<QString> getTopics(const QSet<QString>& message_types, const QSet<QString>& message_sub_types, const QList<QString>& transports);
 
   virtual void selectTopic(const QString& topic);
+  virtual void selectcwRotation(const QString& rotation);
 
 protected slots:
 
   virtual void onTopicChanged(int index);
+  virtual void onRotationChanged(int index);
 
-  virtual void onZoom1(bool checked);
+  virtual void onZoom();
```

Comment:
I did not see anywhere being used. Unless you referring to other forks. I did put it back.